cirq-core/cirq/devices/superconducting_qubits_noise_properties.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass, field
+from typing import Dict, TYPE_CHECKING, List, Set
+
+from cirq import ops, protocols, devices
+from cirq.devices.noise_utils import (
+    OpIdentifier,
+    decoherence_pauli_error,
+)
+
+if TYPE_CHECKING:
+    import cirq
+
+SINGLE_QUBIT_GATES: Set[type] = {
```

Comment:
Since these are copied from the internal version, they're focused on the gates that Google hardware uses. That said, given that this type is _not_ Google-specific, I'll go ahead and add these in (including the other comments below).

Reply:
On second thought, it may be cleaner to make this class more generic, and require subclasses (e.g. the Google-specific version) to specify their gates. That way we don't need to cram every possible gate into these lists, which simplifies things a bunch.